tests/by-util/test_shuf.rs

```diff
@@ -10,7 +10,7 @@ fn test_output_is_random_permutation() {
     let input_seq = vec![1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
     let input = input_seq
         .iter()
-        .map(|x| x.to_string())
+        .map(std::string::ToString::to_string)
```

Comment:
We could import this instead of qualifying it, but it's fine either way.

Reply:
import isn't even needed :)
i cleaned it